.github/workflows/nightly-tests.yml

```diff
@@ -12,4 +12,4 @@ jobs:
 
       - name: Run Tests
         run: |
-          docker run --rm -v `pwd`/models:/models unifyai/ivy-models:latest python3 -m pytest ivy_models_tests/
+          docker run --rm -v `pwd`/models unifyai/models:latest python3 -m pytest ivy_models_tests/
```

Comment:
I think we need to keep `-v 'pwd'/models:/models` instead of `-v 'pwd':/models` right? We don't `cd` into models and therefore we need to include the local `models` path explicitly

Reply:
It already sets the workdir to models in the Dockerfile https://github.com/unifyai/models/blob/master/Dockerfile#L23, so I don't think it's explicitly required